superset/static/help/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
`help` folder will reside under `superset/assets/` parallel to `images` folder. 
Please move the `help` folder to the specified location. `static` folder is a compiled output, we should not keep anything under it.

Reply:
verify --- make sue after compile out help folder also move to static location